webapp/src/components/backstage/public_private_selector.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import {useIntl} from 'react-intl';
+
+import styled from 'styled-components';
+
+type Props = {
+    public: boolean
+    setPlaybookPublic: (pub: boolean) => void
+    disableOtherOption: boolean
+}
+
+const HorizontalContainer = styled.div`
+	display: flex;
+	flex-direction: horizontal;
+	height: 70px;
+`;
+
+const BigButton = styled.button`
+	padding: 16px;
+	border: 1px solid rgba(var(--button-bg-rgb), 0.08);
+	background: var(--center-channel-bg);
+	box-shadow: 0px 2px 3px rgba(0, 0, 0, 0.08);
+	border-radius: 4px;
+	flex-grow: 1;
+	flex-basis: 0;
+	margin: 4px;
+
+    &:disabled {
+        background: rgba(var(--center-channel-color-rgb), 0.08);
+    }
+
+	display: flex;
+	flex-direction: horizontal;
+	align-items: center;
+`;
+
+const StackedText = styled.div`
+	flex-grow: 1;
+	text-align: left;
+`;
+
+const GiantIcon = styled.i<{active?: boolean}>`
+	font-size: 36px;
+	line-height: 42px;
+	color: ${(props) => (props.active ? 'rgb(var(--button-bg-rgb))' : 'rgba(var(--center-channel-color-rgb), 0.56)')};
```

Comment:
We can simplify this too
```suggestion
	color: ${(props) => (props.active ? 'var(--button-bg)' : 'rgba(var(--center-channel-color-rgb), 0.56)')};
```